Marlin/src/pins/pins.h

```diff
@@ -527,16 +527,16 @@
 #elif MB(ARMED)
   #include "stm32/pins_ARMED.h"                 // STM32F4                                env:ARMED
 #elif MB(RUMBA32)
-  #include "stm32/pins_RUMBA32.h"               // STM32F4                                env:RUMBA32
+  #include "stm32/pins_RUMBA32.h"               // STM32F4                                env:rumba32_f446ve env:mks_rumba32
```

Comment:
What can you tell me about the differences between these? We can add a selector to `auto_build.py` for these.

Reply:
I did a little more digging and it appears that there is a RUMBA32 from Aus3D and one from MKS.  

I'll do some more leg work on this.

Reply:
Speaking of `auto_build.py` is it acting any differently for you lately? My python penv might be sketchy or something, because the Tk window is not painting correctly, and the build reverted to happening in an extra custom Tk window. Otherwise it works.

Not to sweat it. I'm working on replacing the Python with Javascript and moving the interface into a "Home Page" so before you initiate the build all the info and options are listed in a nice web view.

Reply:
The two boards have the same pinout and almost identical layout.

MKS grabbed the AUS design and then added some LEDs and pull up resistors and switched to different power FETs.

The Aus3D version is available from multiple sources.  I didn't see the MKS version for sale and it's not mentioned on the Makerbase web site.

I'll do a PR to add the MKS flavor and document the differences.